_src/static/sass/utilities/_helpers.scss

```diff
@@ -0,0 +1,139 @@
+// ------------------------------
+// edX Pattern Library: Helpers
+
+// About: Sass partial for defining extends and re-usable elements at the Sass compile level.
+
+// #UTILITIES
+// #RESETS
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+// visual link
+%fake-link {
+  cursor: pointer;
+}
+
+// visual disabled
+%state-disabled {
+  pointer-events: none;
+  outline: none;
+  cursor: default;
+}
+
+// presentational wrapper
+%wrapper {
+    @include clearfix();
+    width: 100%;
+}
+
+// remove spacing from first/last children
+%clear-first-child {
+
+    &:first-child {
+        margin-top: 0;
+        padding-top: 0;
+        border-top: none;
+    }
+}
+
+%clear-last-child {
+
+    &:last-child {
+        margin-bottom: 0;
+        padding-bottom: 0;
+        border-bottom: none;
+    }
+}
+
+// accessibility
+%a11y-hide {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
+
+%a11y-hide-image-text {
+    overflow: hidden;
+    text-indent: 100%;
+    white-space: nowrap;
+}
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+// reset of lists
+%no-list {
```

Comment:
Is `no-list` accurate? This would go _on_ a list, no? I feel like this lacks semantics.